dump.rdb

```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```

Comment:
Stray file. You could add dump.rdb to the git ignore file, if you like, or just remove it from the commit.